pom.xml

```diff
@@ -54,8 +54,6 @@
         <module>lib/trino-memory-context</module>
         <module>lib/trino-orc</module>
         <module>lib/trino-parquet</module>
-        <!-- TODO(https://github.com/trinodb/trino/issues/13051): Remove whole module when Phoenix5 is released -->
-        <module>lib/trino-phoenix5-patched</module>
```

Comment:
Don't you need to remove https://github.com/trinodb/trino/tree/master/lib/trino-phoenix5-patched files?

Reply:
Yep. But it doesn't work anyway so I won't be able to merge this PR. Phoenix gets stuck randomly.

Reply:
Sorry, I have been silent here. I do not work on/with Phoenix anymore.
That said, how does it get stuck? Is there an easy way to reproduce?

Reply:
```
testing/trino-product-tests-launcher/bin/run-launcher test run --config config-default \
--environment multinode-phoenix5 \
-- -g configured_features,phoenix
```

will sometimes hang and sometimes succeed